### PR TITLE
Crashed by apostrophe

### PR DESCRIPTION
**Pull request by rendo0m**
Hello David,
the [pynab project](https://github.com/nabaztag2018/pynab) comes with comitup. 
Very easy if you don't own or having a wifi with an "apostrophe" in your range.
Maybe you can review my changes, that made everything work for me. :-)
Thanks for your work!

### CODE REVIEW

File: comitup/iwscan.py [hunk 1]
```diff
@@ -83,7 +84,7 @@ def devaps(dev: str, dump: str = "") -> List[Dict[str, str]]:
             ap = blk2dict(blk)
             ap["power"] = dbm2pct(float(ap["signal"].split()[0]))
 
-            if ap["SSID"] and ap["SSID"].count(r"\x00") == 0:
+            if ap["SSID"] and ( ap["SSID"].count(r"\x00") == 0 or ap["SSID"].count(r"'") == 0 ):
```
Comment (davesteele) [inline comitup/iwscan.py:87]: This part doesn't look right. It is supposed to hide APs that are supposed to be hidden (suggesting that the terms should be AND'd). Does your fix work without this change?

**Comment by wspi** [comment]
That didn't fix #212 

I'll try some code changes due to the SSID with apostrophe to see if it's the same issue, or try in a place where this SSID is out of range to verify if it's the same issue

**Comment by wspi** [comment]
after some tests, it did indeed worked, not sure if it was another SSID that it's not present anymore, but there is one SSID with apostrophe and applying this changes worked, although I've tests without the change in line 87 and it's still working after reboot

**Comment by davesteele** [comment]
Thanks to both of you.

**Pull request merged**